__tests__/integration/previewRDFHelper.js

```diff
@@ -27,3 +27,18 @@ export async function fillInRequredFieldsForBibframeInstance() {
   await page.waitForSelector('#rbt-menu-item-0')
   await page.click('#rbt-menu-item-0')
 }
+
+export async function incompleteFieldsForBibframeInstance() {
+  // This assertion adds 1 to each it blocks assertion count
+  await pupExpect(page).toClick('a', { text: 'BIBFRAME Instance' })
+
+  // Click on one of the property type rows to expand a nested resource
```

Comment:
do we need the rest of this function?

Reply:
Oh yeah, probably not. That's just how I was manually testing it. Will update.

Reply:
Done. Good catch.